api/resolvers/item.js

```diff
@@ -28,6 +28,8 @@ import { verifyHmac } from './wallet'
 import { parse } from 'tldts'
 import { shuffleArray } from '@/lib/rand'
 
+metadataRuleSets.title.rules.unshift(['h1 > yt-formatted-string.ytd-watch-metadata', el => el.getAttribute('title')])
```

Comment:
I don't think this should have been added here as top-level code.

Maybe that's related to https://github.com/stackernews/stacker.news/issues/2164#issuecomment-3042286949.

Reply:
Or nvm, I don't think it's related but I don't like to have this here as top-level code. I think we could have done in a variable local to `pageTitleAndUnshorted`.